lib/extract/netcdf/lfric/Makefile

```diff
@@ -71,7 +71,7 @@ LFRIC_INCLUDE_FLAGS += $$(nf-config --fflags)
 PSYDATA_LIB_NAME = _extract
 PSYDATA_LIB = lib$(PSYDATA_LIB_NAME).a
 
-PROCESS_ARGS = -prefix=extract_ -types=char,int,logical,real,double \
```

Comment:
Is this whole folder needed anymore? The type and dims are the same as the parent Makefile, the jinja files have the implementation with `%get_proxy() ` but the lowering version may not need this? and for the prefix we could be consistent with the generic. Is there something else?

If it can be removed, maybe for a separate PR.

Reply:
I've opened #3044 for that. It affects quite a few files and documentation.